unison-src/transcripts/emptyCodebase.output.md

```diff
@@ -9,33 +9,6 @@ BEHOLD!!!
 ```ucm
 .> ls
 
-  nothing to show
-
-```
-Technically, the definitions all exist, but they have no names. `builtins.merge` brings them into existence, under the current namespace:
-
-```ucm
-  ☝️  The namespace .foo is empty.
-
-.foo> builtins.merge
-
-  Done.
-
-.foo> ls
-
-  1. builtin/ (328 definitions)
-
-```
-And for a limited time, you can get even more builtin goodies:
-
-```ucm
-.foo> builtins.mergeio
-
-  Done.
-
-.foo> ls
-
-  1. builtin/ (491 definitions)
-
-```
-More typically, you'd start out by pulling `base.
+  ⚠️
+  
+  The namespace . doesn't exist.
```

Comment:
This one is weird - the transcript just [cuts out](https://raw.githubusercontent.com/unisonweb/unison/454fb392e81e5423cbce654931aef1956ce8f642/unison-src/transcripts/emptyCodebase.output.md). It doesn't even close the backticks of [that first output block](https://raw.githubusercontent.com/unisonweb/unison/454fb392e81e5423cbce654931aef1956ce8f642/unison-src/transcripts/emptyCodebase.md). 

@runarorama If I had to guess, it looks like the transcript marks that first block as `ucm:error`, but the new output for `ls` of an empty namespace isn't considered an error. You can look at `Codebase.Editor.Output`, the `isFailure` function, and make sure that `ls` of an empty namespace is considered an error by your branch.